IBF-Typhoon-model/scr/typhoon_model/pipeline.py

```diff
@@ -317,27 +320,25 @@ def main(path,remote_directory,typhoonname):
             #image_filename=landfall_typhones[0]
             image_filename=[i for i in landfall_typhones if i.endswith('.png')][0]
             data_filename=[i for i in landfall_typhones if i.endswith('.csv')][0]
+            data_filename1=[i for i in landfall_typhones if i.endswith('.csv')][1]
```

Comment:
It would be a bit clearer if you named these `data_filename1` and `data_filename2`